client/components/trip/itineraryTrip.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
```

Comment:
does not seem to be updating live across 2 instances on my end, no way to add itinerary to an existing trip